### PR TITLE
Add rating aggregates to the add-on review list API, if requested.

### LINKED ISSUE #3550: Add API to return ratings aggregates for an add-on (or add it to existing reviews list API)
We need an API returning an aggregate of the number of users that gave each rating score (1-5). Or maybe just add it to the reviews list API if a specific parameter is passed.

It should be pretty straightforward, just need to use `GroupedRating.get(addon_id)`.

Supports #3024

### PR DESCRIPTION
**Pull request by diox**
Fix #3550

### CODE REVIEW

File: docs/topics/api/reviews.rst [hunk 1]
```diff
@@ -19,10 +19,12 @@ This endpoint allows you to fetch reviews for a given add-on.
 .. http:get:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/
 
     :query string filter: The :ref:`filter <review-filtering-param>` to apply.
+    :query int show_grouped_ratings: Whether or not to show ratings aggregates for this add-on in the response.
     :>json int count: The number of results for this query.
     :>json string next: The URL of the next page of results.
     :>json string previous: The URL of the previous page of results.
     :>json array results: An array of :ref:`reviews <review-detail-object>`.
+    :>json object grouped_ratings: An object aggregating the number of times each rating has been posted for this add-on. Only present if ``show_grouped_ratings`` query parameter is present.
```
Comment (kumar303) [inline docs/topics/api/reviews.rst]: For this case, I think an example of the object (in JSON) would clarify it. I forget if the ReST parameter markup allows you to add a code block in the parameter description, I'm pretty sure it does.

File: src/olympia/reviews/views.py [hunk 2]
```diff
@@ -392,6 +392,14 @@ def filter_queryset(self, qs):
                 raise ParseError('Need an addon or user identifier')
         return qs
 
+    def get_paginated_response(self, data):
+        response = super(ReviewViewSet, self).get_paginated_response(data)
+        if ('addon_pk' in self.kwargs and
+                self.request.GET.get('show_grouped_ratings')):
```
Comment (EnTeQuAk) [inline src/olympia/reviews/views.py]:
```python
show_grouped_ratings = self.request.GET.get('show_grouped_ratings')
if 'addon_pk' in self.kwargs and show_grouped_ratings:
    # ...
```

should fit on one line and is easier to scan imho.

File: docs/topics/api/reviews.rst (hunk 1 above)
Comment (diox) [inline docs/topics/api/reviews.rst]: Fixed

File: src/olympia/reviews/views.py (hunk 2 above)
Comment (diox) [inline src/olympia/reviews/views.py]: Fixed

**Pull request merged**